src/app/DTT/rating/Rating.js

```diff
@@ -5,8 +5,8 @@ const Rating = ({ value, hover, setHover, onChange, questionCode }) => {
   if (!setHover)
     return (
       <HStack spacing="0">
-        {[1, 2, 3, 4, 5].map(idx => (
-          <div key={idx} style={{ width: '2rem', height: '2rem', padding: '0.5rem' }}>
+        {[1, 2, 3, 4, 5].map((idx, index) => (
+          <div key={index} style={{ width: '2rem', height: '2rem', padding: '0.5rem' }}>
```

Comment:
Should not use index as a key, this can introduce bugs such as the wrong UI being displayed. Also not efficient with how mapping works. Use other ways of generating unique keys.

Reply:
Thanks. I'll use other keys.